app/components/admin/custom_fields/hierarchy/items_component.html.erb

```diff
@@ -53,23 +61,14 @@ See COPYRIGHT and LICENSE files for more details.
               box.with_footer(test_selector: "op-custom-fields--new-item-form") do
                 render Admin::CustomFields::Hierarchy::ItemFormComponent.new(
                   target_item: @new_item,
-                  url: new_child_custom_field_item_path(model.root.custom_field_id, model),
+                  url: new_child_custom_field_item_path(root.custom_field_id, model),
```

Comment:
Can we use the `new_item_path` here as well?

Reply:
Oooh... I knew I missed something. :P